docker/Readme.md

```diff
@@ -0,0 +1,15 @@
+# Docker in PCF
+
+If you're using docker in PCF, you can't use the buildpack. This and the accompinying dockerfile will provide a guide to adding something very similar to this buildpack to your docker images.
+
+## Dockerfile
+
+The dockerfile contains both the datadog assets as well as a small flask app, and shows you how to set up an entrypoint file to start all of these things
+
+## Flask app
+
+The application is a simple flask application. It contains both tracing, dogstatsd and logs. The requirements.txt file contians the requirements
+
+## Entrypoint
+
+The entrypoint file has the two datadog scripts sourced, instead of them being run directly. They need to be sourced because sourcing them means that they aren't blocking on the main script.
```

Comment:
```suggestion
The entrypoint file has the two Datadog scripts sourced, instead of them being run directly. Having them sourced means that they aren't blocking on the main script.
```